lib/vanagon/platform/windows.rb

```diff
@@ -196,6 +172,7 @@ def initialize(name)
         @num_cores = "/usr/bin/nproc"
         @install = "/usr/bin/install"
         @copy = "/usr/bin/cp"
+        @package_type = "msi"
```

Comment:
Nope, I think we should leave it like this. People should be deliberate about the type of package they are building and testing. They can remove the line once we have the MSI work done.